docs-ui/components/activity.stories.js

```diff
@@ -17,95 +16,99 @@ const user = {
   email: 'billy@sentry.io',
 };
 
-storiesOf('UI|Activity/Item', module)
-  .add(
-    'default ActivityItem',
-    withInfo(
-      'An Activity Item is composed of: an author, header, body, and additionally timestamp and a status.'
-    )(() => (
-      <ActivityItem
-        author={{type: 'user', user}}
-        item={{id: '123'}}
-        date={new Date()}
-        header={<div>{user.email}</div>}
-        hideDate={boolean('Hide Date', false)}
-      >
-        Activity Item
-      </ActivityItem>
-    ))
-  )
-  .add(
-    'with custom Header',
-    withInfo('Activity Item with a custom header')(() => (
-      <ActivityItem
-        author={{type: 'user', user}}
-        item={{id: '123'}}
-        date={new Date()}
-        header={() => (
-          <div style={{backgroundColor: '#ccc'}}>Custom header (no timestamp)</div>
-        )}
-      >
-        Activity Item
-      </ActivityItem>
-    ))
-  )
-  .add(
-    'with footer',
-    withInfo('Activity Item with a footer')(() => (
-      <ActivityItem
-        author={{type: 'user', user}}
-        item={{id: '123'}}
-        date={new Date()}
-        hideDate={boolean('Hide Date', false)}
-        header={<div>{user.email}</div>}
-        footer={<div>Footer</div>}
-      >
-        Activity Item
-      </ActivityItem>
-    ))
-  )
-  .add(
-    'system activity',
-    withInfo('An ActivityItem generated by Sentry')(() => (
-      <ActivityItem
-        author={{type: 'system'}}
-        item={{id: '123'}}
-        date={new Date()}
-        header={<div>Sentry detected something</div>}
-        hideDate={boolean('Hide Date', false)}
-      >
-        Sentry did something
-      </ActivityItem>
-    ))
-  );
+export default {
+  title: 'UI/Activity/Item',
+};
+
+export const DefaultActivityItem = withInfo(
+  'An Activity Item is composed of: an author, header, body, and additionally timestamp and a status.'
+)(() => (
+  <ActivityItem
+    author={{type: 'user', user}}
+    item={{id: '123'}}
+    date={new Date()}
+    header={<div>{user.email}</div>}
+    hideDate={boolean('Hide Date', false)}
+  >
+    Activity Item
+  </ActivityItem>
+));
+
+DefaultActivityItem.story = {
+  name: 'default ActivityItem',
```

Comment:
Do you really prefer to display "default" in lowercase? I think it would be fine displaying "Default Activity Item" no?